pkg/mtq-controller/vmmrq-controller/vmmrq-controller.go

```diff
@@ -195,6 +195,38 @@ func (ctrl *VmmrqController) enqueueMigration(obj interface{}) {
 	ctrl.migrationQueue.Add(key)
 }
 
+func (ctrl *VmmrqController) enqueueAll() {
+	vmmrqObjs := ctrl.vmmrqInformer.GetIndexer().List()
+	for _, vmmrqObj := range vmmrqObjs {
+		vmmrq := vmmrqObj.(*v1alpha12.VirtualMachineMigrationResourceQuota)
+		if vmmrq.Status.OriginalBlockingResourceQuotas == nil || len(vmmrq.Status.OriginalBlockingResourceQuotas) == 0 {
+			continue
+		}
+		atLeastOneMigration := false
+		for blockedMigrationName := range vmmrq.Status.MigrationsToBlockingResourceQuotas {
+			ctrl.migrationQueue.Add(vmmrq.Namespace + "/" + blockedMigrationName)
+			atLeastOneMigration = true
+		}
+		if !atLeastOneMigration { //should update vmmrq.status even if there aren't any blocked migrations
+			ctrl.migrationQueue.Add(vmmrq.Namespace + "/fake")
+		}
+	}
```

Comment:
if len(vmmrq.Status.MigrationsToBlockingResourceQuotas) != 0 {
		for blockedMigrationName := range vmmrq.Status.MigrationsToBlockingResourceQuotas {
			ctrl.migrationQueue.Add(vmmrq.Namespace + "/" + blockedMigrationName)
			atLeastOneMigration = true
		} else {
		        //should update vmmrq.status even if there aren't any blocked migrations
			ctrl.migrationQueue.Add(vmmrq.Namespace + "/fake")
		}
	}

Reply:
Done